.github/workflows/build.yml

```diff
@@ -34,6 +34,40 @@ jobs:
       uses: codecov/codecov-action@v1
       with:
         fail_ci_if_error: true
+    
+
+  lint:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.8
+    - name: Install Dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install pipenv
+        pipenv install --dev
+    - name: Check with black
+      run: |
+        pipenv run black aguaclara --check
+    - name: If check failed, autoformat using black and push
+      if: failure()
+      run : |
+        # taken from https://github.com/psf/black/actions/runs/17913292/workflow
+        pipenv run black aguaclara 
+        git config --global user.name 'autoblack'
+        git config --global user.email 'annabel48lin@users.noreply.github.com'
```

Comment:
I think you can avoid using a personal account with a bot account like in the [docs workflow](https://github.com/AguaClara/aguaclara/blob/master/.github/workflows/docs.yml#L36-L37). But hopefully that isn't necessary with a pre-packaged action for using `black`!